scenes/game/local_game.gd

```diff
@@ -2531,6 +2550,11 @@ func handle_strike_effect(card_id :int, effect, performing_player : Player):
 		"armorup":
 			performing_player.strike_stat_boosts.armor += effect['amount']
 			events += [create_event(Enums.EventType.EventType_Strike_ArmorUp, performing_player.my_id, effect['amount'])]
+		"armorup_damage_dealt":
+			## This is inaccurate if you have this effect on an additional attack character -- currently impossible, and additional attacks mess up many things
```

Comment:
Just one # for comments.
Can you explain exactly what is inaccurate?
I guess we haven't implemented additional attacks, but I would expect get_damage_taken to return the total damage of all multiple strikes?

Reply:
The correct logic for this card only gets armor for the damage that it itself does, but if this were played as an additional attack it would get incorrect amounts of armor since get_damage_taken works the way you're describing. Will try to make comment clearer, don't imagine this situation will ever be possible even in a new set.